app/assets/javascripts/admin/addon/components/schema-theme-setting/types/tags.gjs

```diff
@@ -0,0 +1,65 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { and, not } from "truth-helpers";
+import I18n from "discourse-i18n";
+import FieldInputDescription from "admin/components/schema-theme-setting/field-input-description";
+import TagChooser from "select-kit/components/tag-chooser";
+
+export default class SchemaThemeSettingTypeTag extends Component {
```

Comment:
Hmm I'm not sure if you want to pluralise this. 

Like, tags chooser vs tag chooser.

Reply:
O yea I should now that it has been renamed.